app/pages/lab-fem/components/fem-multi-image-subject-options-editor.jsx

```diff
@@ -68,6 +76,17 @@ export default function FemMultiImageSubjectLayoutEditor ({
           Allow Separate Frames View - <small>volunteers can choose flipbook or a separate frames view</small>
         </label>
       </div>
+      
+      <div>
+        <label>
+          <input
+            type='checkbox'
+            checked={cloneMarksChecked}
+            onChange={toggleCloneMarks}
+          />
+          Clone markers in all frames - <small>for drawing tasks</small>
```

Comment:
```suggestion
          Clone marks in all frames - <small>for drawing tasks</small>
```

NB suggestion - we tend to call drawing tasks "marks" in the code and also other places in the lab so this could be a good opportunity for standardization here too, at least for the fem lab.